### PR TITLE
Use the oc CLI tool to match the rest of the repo documentation.

### PR DESCRIPTION
**Pull request by adelton**
## Description

Use the oc CLI tool to match the rest of the repo documentation.

## How Has This Been Tested?

Manually.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ MAKE_ENV_FILE = local.vars.mk
 
 # Setup the secret required for observability then generate and apply the files
 setup-observability: acm/odh-core/acm-observability/kustomization.yaml
-	kustomize build acm/odh-core/acm-observability | kubectl apply -f -
+	oc kustomize acm/odh-core/acm-observability | oc apply -f -
```
Comment (grdryn) [inline Makefile:12]: I guess this one could alternatively be `oc apply -k acm/odh-core/acm-observability`

**Review by grdryn — APPROVED** [review]
/lgtm

I didn't realise there was an `oc kustomize` sub-command 👍

File: Makefile (hunk 1 above)
Comment (adelton) [reply Makefile]: Updated, rebased on `main`. Sorry it took me a while to get back to this.

**Review by MarianMacik — APPROVED** [review]
/approve

**Review by grdryn — APPROVED** [review]
/lgtm

**Pull request merged**